package.json

```diff
@@ -30,5 +30,9 @@
       "<rootDir>/packages/*",
       "<rootDir>/test-packages/"
     ]
+  },
+  "volta": {
+    "node": "10.16.1",
```

Comment:
This version was chosen, as it seems to be compatible with the version of native extensions locked in the yarn file. We can bump this when the lockfiles are re-rolled

Reply:
https://github.com/embroider-build/embroider/pull/290 aims to re-roll the lockfile